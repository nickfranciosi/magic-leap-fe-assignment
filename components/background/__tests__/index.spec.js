/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Background from '../index';

describe('Background', () => {
  test('it should render', () => {
    const tree = renderer.create(<Background />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('it should fill the view width', () => {
    const tree = renderer.create(<Background />).toJSON();
    expect(tree).toHaveStyleRule('width', '100vw');
  });
});
