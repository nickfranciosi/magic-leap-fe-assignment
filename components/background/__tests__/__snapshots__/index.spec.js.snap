// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background it should render 1`] = `
.c0 {
  position: fixed;
  top: 0;
  width: 100vw;
  height: 100vh;
  z-index: -1;
  background-color: #151515;
}

.c2 {
  position: absolute;
  width: 150px;
  height: 100%;
  top: 0;
  background-size: 100% auto;
  background-repeat: repeat-y;
}

.c1 {
  left: 0;
  background-position: left center;
  background-image: url('/static/images/bg_stars_left.jpg');
}

.c3 {
  right: 0;
  background-position: right center;
  background-image: url('/static/images/bg_stars_right.jpg');
}

<div
  className="c0"
>
  <div
    className="c1 c2"
  />
  <div
    className="c3 c2"
  />
</div>
`;
